yt/yql/tests/test_simple.py

```diff
@@ -42,6 +43,47 @@ def _test_simple_query(self, query, expected, **kwargs):
 
         assert_items_equal(result, expected)
 
+    def _exists_pending_stage_in_progress(self, query):
+        queryInfo = query.get()
+        if "progress" in queryInfo and "yql_progress" in queryInfo["progress"]:
+            for stage in queryInfo["progress"]["yql_progress"].values():
+                if "pending" in stage and stage["pending"] > 0 :
+                    return True
+        return False
+
+
+class TestMetrics(TestQueriesYqlBase):
+    @authors("mpereskokova")
+    def test_metrics(self, query_tracker, yql_agent):
+        create("table", "//tmp/t", attributes={
+            "schema": [{"name": "a", "type": "int64"}]
+        })
+        rows = [{"a": 42}]
+        write_table("//tmp/t", rows)
+
+        create_pool("small", attributes={"resource_limits": {"user_slots": 0}})
+        query = start_query("yql", 'pragma yt.StaticPool = "small"; select a+1 as result from primary.`//tmp/t`')
+
+        wait(lambda: self._exists_pending_stage_in_progress(query))
+
+        def filter_sensor(sensor):
+            return sensor["labels"]["sensor"] == "yt.yql_agent.active_queries"
+
+        def lastActiveQueriesMetric():
+            sensors = requests.get(f"http://localhost:{yql_agent.yql_agent.configs[0]["monitoring_port"]}/solomon/shard/default").json()["sensors"]
```

Comment:
you can take a look at profiler_factory() in the other integration tests for boilerplate elimination